src/main/kotlin/com/dp/advancedgunnerycontrol/typesandvalues/WeaponAITags.kt

```diff
@@ -34,6 +33,7 @@ fun shouldTagBeDisabled(groupIndex: Int, sh: FleetMemberAPI, tag: String) : Bool
 
 val tagTooltips = mapOf(
     "PD" to "Restricts targeting to fighters and missiles.",
+    "PrioritisePD" to "Weapon will always prioritise from small to large (Missiles > fighters > small ships > big ships)",
     "NoPD" to "Forbids targeting missiles and prioritizes ships over fighters.",
     "Fighter" to "Restricts targeting to fighters.",
     "AvoidShields" to "Weapon will prioritize targets without shields, flanked shields or high flux/shields off.",
```

Comment:
Would be awesome to add a phrase like "Shields of fighters will [not] be ignored (configurable in settings)" to all the shield-modes, depending on whether the corresponding setting is enabled.

Reply:
Good idea, added this :)